core/app/assets/js/loader/commands/close.js

```diff
@@ -6,8 +6,8 @@ export class Close extends CommandBase {
 			return false;
 		}
 
-		this.component.iframe.style.display = 'none';
-		document.body.style.overflow = '';
+		this.component.iframe.remove();
```

Comment:
Route to the default screen instead of having to reload the whole content

Reply:
Currently the editor and the app are not fully synchronized, so, we need a full reload for cases that the data has been changed by the editor.